src/control/control.h

```diff
@@ -13,6 +13,15 @@
 
 class ControlObject;
 
+enum class ControlFlag {
+    None = 0,
+    NoWarnIfMissing = 1,
```

Comment:
The negation make the code harder to understand. Why not `WarnIfMissing` and `WarnAndAssertIfMissing`? Using flags is not necessary according to the code, because the assertion is nested in the warning path. Use a simple enum class instead.

Reply:
I was planning on adding a few more flags, e.g. `DelayedInitialization` for ControlProxy objects that are created before the actual CO (e.g. `[AutoDJ],enabled`). Right now we just disable the warnings completely, but IMHO this is not a good solution.

Reply:
> Why not WarnIfMissing and WarnAndAssertIfMissing?

I wanted to make it explicit from the caller side that the warnings/assertions are disabled, so that don't set `InitializeLater` and accidently disable warnings/assertions because you forgot to OR this with the `WarnAndAssertIfMissing` flag.

Btw, I also had to add a flag to allow missing keys for WWidgetStack (which takes params from skins).